packages/store/src/IFieldLayoutErrors.sol

```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.24;
+
+/**
+ * @title IFieldLayoutErrors
+ * @author MUD (https://mud.dev) by Lattice (https://lattice.xyz)
+ * @notice This interface includes errors for the FieldLayout library.
+ */
```

Comment:
Can we add the _why_ to each of these?

```suggestion
/**
 * @title IFieldLayoutErrors
 * @author MUD (https://mud.dev) by Lattice (https://lattice.xyz)
 * @notice This interface includes errors for the FieldLayout library.
 * @dev We centralize and reference these errors in an interface so that we can ensure they're included in the resulting Store ABI for proper decoding in the frontend. This is a workaround until the Solidity compiler will do this for us.
 */
```